py/tests/plane_move.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python3
+
+import pykms
+import random
+import time
+import sys
+import select
+import selectors
+
+if len(sys.argv) not in (1, 3):
+    print('Without parameters: list of available connectors')
+    print('{} [name of connector0] [name of connector1]\n'.format(sys.argv[0]))
+    sys.exit(-1)
+
+card = pykms.Card()
+
+if not card.has_atomic:
+    print('Atomic mode settings is not supported :(')
+    sys.exit(-1)
+
+if len(sys.argv) == 1:
+    print('List of available connectors:')
+    for conn in card.connectors:
+        if conn.connected() == 1:
+            mode = conn.get_default_mode()
+            print(' {}: {} ({}x{})'.format(conn.idx, conn.fullname,
+                                           mode.hdisplay, mode.vdisplay))
+    print('\nNow you can try:')
+    print('{} [name of connector0] [name of connector1]\n'.format(sys.argv[0]))
+    sys.exit()
+
+res = pykms.ResourceManager(card)
+
+conn_list = []
+crtc_list = []
+mode_list = []
+rootplane_list = []
+fb_list = []
+colors = []
+
+src_w = 300
+src_h = 300
+
+for i in range(2):
+    conn = res.reserve_connector(sys.argv[i + 1])
+    if conn.connected() == 1:
```

Comment:
I believe it returns a boolean, not integer

Reply:
conn can be None, if not found. Instead of not adding it to a list, you should just exit with an error. And then the test conn_list test below is not needed.

Reply:
OK